account_banking_pain_base/models/account_payment_order.py

```diff
@@ -105,6 +111,8 @@ def _sepa_iban_prefix_list(self):
         ]
 
     @api.depends(
+        "payment_mode_id.payment_method_id.pain_version",
```

Comment:
I don't think you need this convoluted depends. Only `payment_mode_id`. If not, changing these fields in payment methods will trigger the recomputation of all payment orders, which is not performant and even not convenient.

Reply:
done